src/EnergyPlus/Furnaces.cc

```diff
@@ -5034,6 +5029,8 @@ namespace Furnaces {
 		// Check ventilation/fan load for constant fan systems to see if load to be met changes
 		// Same IF logic used in Subroutine SetAverageAirFlow to determine if unit is ON or OFF
 
+		QToCoolSetPt = 0.0;
+		QToHeatSetPt = 0.0;
```

Comment:
I have to ask why. The important return variable is ZoneLoad. If the code executes the below code for constant fan, these will be set. These aren't used otherwise. If there are 2 air loops, won't these variables represent what the 2nd air loop must meet?

Reply:
I'll revisit these.  I added these because something complained about using these uninitialized - either a build warning or a test failure (don't remember which).